ledger/accountdb.go

```diff
@@ -79,6 +79,13 @@ var accountsSchema = []string{
 		strval text)`,
 }
 
+// TODO: Post applications, rename assetcreators -> creatables and rename
+// 'asset' column -> 'creatable'
+var creatablesMigration = []string{
+	`ALTER TABLE assetcreators ADD COLUMN ctype INTEGER DEFAULT 0`,
+	`CREATE INDEX assetcreators_ctype_idx ON assetcreators (asset, ctype)`,
```

Comment:
Out of curiosity, how much does the index help?

Reply:
I think this is just there for the `v1` `listAssets` endpoint. Any benefit is likely to be pretty marginal -- I would be in favor of getting rid of this entirely unless @algorandskiy objects. Easy to add back if some kind of perf issue pops up.

Reply:
The specific query is `SELECT asset, creator, ctype FROM assetcreators WHERE asset <= ? AND ctype = ? ORDER BY asset desc LIMIT ?`

Reply:
removed